DigitalOffice/Services/CheckRightsService/Database/CheckRightsServiceDbContext.cs

```diff
@@ -0,0 +1,40 @@
+﻿using CheckRightsService.Database.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace CheckRightsService.Database
+{
+    /// <summary>
+    /// Represents database for rights checking.
+    /// </summary>
+    public class CheckRightsServiceDbContext : DbContext
+    {
+        public DbSet<Right> Rights { get; set; }
+
+        public DbSet<RightChangeRecord> RightsHistory { get; set; }
+
+        public DbSet<RightProjectLink> RightProjectLinks { get; set; }
+
+        public CheckRightsServiceDbContext(DbContextOptions<CheckRightsServiceDbContext> options) : base(options) { }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
Please use [separate configuration classes](https://www.learnentityframeworkcore.com/configuration/fluent-api#separate-configuration-classes). Put config class in the same file where model is.

Reply:
> Please use [separate configuration classes](https://www.learnentityframeworkcore.com/configuration/fluent-api#separate-configuration-classes). Put config class in the same file where model is.

Added config classes

Reply:
use ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());